### ISSUE TITLE
main.py not recognizing module reddit.askreddit

### ISSUE THREAD

**Issue by FreebieII** [elebumm/RedditVideoMakerBot#49]
`from reddit.askreddit import get_askreddit_threads`

**ModuleNotFoundError: No module named 'reddit.askreddit'**

![image](https://user-images.githubusercontent.com/69956546/171386315-a1fb6843-769c-4ee7-8984-c9a61164b337.png)

**Comment by BlockArchitech** [comment]
Are all the files in the same place? That import line is pointing towards `reddit/adkreddit.py`

**Comment by FreebieII** [comment]
> Are all the files in the same place? That import line is pointing towards `reddit/askreddit.py`

Yes, that's where it is, same as the main code, didn't change much in the directory [Ignore the import errors of other packages]

![image](https://user-images.githubusercontent.com/69956546/171398032-7f36c108-4282-410f-add1-d64543cc0d70.png)
![image](https://user-images.githubusercontent.com/69956546/171398049-5edfa404-8df8-4059-89fb-ab9ace7d04fc.png)
![image](https://user-images.githubusercontent.com/69956546/171398093-f3ea5a81-1074-42ca-9518-ca000a8fb4ff.png)
